ui-asset-management/api/svc-asset-details/extension/clients/rds_client.go

```diff
@@ -64,14 +64,16 @@ func NewRdsClient(configuration *Configuration) *RdsClient {
 func (r *RdsClient) GetPluginName(ctx context.Context, source string) (string, error) {
 	println("Getting Plugin name from RDS")
 	query := `
-						 SELECT p.name 
+						 SELECT p.name as pluginName 
```

Comment:
I have added this for handling the legacy asset model. But, since the asset models will be updated after merging the changes for delta engine, I think we can remove it.